tools/cosmovisor/CHANGELOG.md

```diff
@@ -36,6 +36,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
```

Comment:
Time to release a new version of cosmovisor after having done some testing I think.
I will do that next week.